### ISSUE TITLE
Modfying Text within a table

### ISSUE THREAD

**Issue by chriscap-fd** [singerla/pptx-automizer#77]
Are there any examples of modifying text within a table.  Specifically, I'm looking to

1.  Change color to white
2.  Bold
3.  Italicize
4.  Change background color (I believe there ARE samples for this one)

I can see how to do that outside of a table, but couldn't find any examples inside.

**Comment by singerla** [comment]
That's right, there was no example, but now there is. Please note: Applying styles to table cells will work properly if the cells in your template already have a default value for the target style.

I needed to pre-format the template cell in `SlideWithTables.pptx` with sRgb color to make it work.

**Comment by chriscap-fd** [comment]
And I guess one more thing.  Is there magic to getting top borders working?
        const totalStyle = {
        border: [
            {
                tag: 'lnB',
                weight: 10000,
                type: 'solid',
                color: {
                    type: 'srgbClr',
                    value: '000000',
                }
            },
            {
                tag: 'lnT',
                weight: 10000,
                type: 'solid',
                color: {
                    type: 'srgbClr',
                    value: '000000',
                }
            }
        ],
        isBold: true,
    };

Using this code, but getting this result.  "Total Taxable Assets" should have a top border.  The left and right borders work properly as well as the bottom.

![image](https://github.com/singerla/pptx-automizer/assets/100152427/de5535a2-e137-4743-b26c-f45b7ada4f71)

**Comment by singerla** [comment]
Please take a look at the update test file. Font size is easy :smile: 

Regarding border style: You need to style the adjacent cell borders as well. In your case, "Gifting Account" needs a bottom border, too.

Font alignment is not yet implemented.

**Comment by chriscap-fd** [comment]
Thanks for the font size example.  RE: the styling of the border above, is that a PPT limitation or something with the XML that requires you to style both?  I think expected behavior would be that if you style top border, you wouldn't have to simultaneously style bottom border of the cell above.  By this logic, if you wanted a bottom border, you would have to apply a top border on the lower cell.  But it doesn't work that way in this library.  The bottom border just draws.  Same with left and right.  They just draw without needing to put opposite border on.

**Comment by singerla** [comment]
I'm not sure on this. Formatting borders around pptx table cell causes trouble since 2001 :smile: 

You can try this to figure out when, if and probably how it works:

```ts
  const pres = automizer
    .loadRoot(`RootTemplate.pptx`)
    .load(`SlideWithTables.pptx`)
    .addSlide("SlideWithTables.pptx", 3, (slide) => {
      slide.modifyElement("TableWithEmptyCells", [
        // Dump XML before editing
        // modify.dump,
        
        modify.setTable(data1),
        
        // Dump XML after editing
        modify.dump,
      ]);
    });
```

I can remember that it's highly important to add `<a:lnL ... />`, ..., `<a:lnB ... />` elements in the right order to prevent ppt to complain on opening a file.

An improvement would also be to have a style attached directly to a cell when defining data:

```ts
const data1 = { body: { values: ['test3', 14, 12, 11, {
  value: 14,
  style: {
    // this could automatically apply border style to adjacent cells if needed
  }
}], } }
